.github/workflows/build-appimage.yml

```diff
@@ -0,0 +1,81 @@
+name: Build FLINT AppImage
+on:
+  push:
+    tags:
+      - 'v*'
+
+jobs:
+  build:
+    strategy:
+      matrix:
+        os: [ubuntu-18.04, ubuntu-20.04, ubuntu-22.04]
+
+    runs-on: ${{ matrix.os }}
+
+    steps:
+      - name: Clone and run the AppImage script
+        run: |
+          git clone https://github.com/ankitaS11/AppImageDataForFlint
```

Comment:
Migrate all these scripts to this repository :)

Reply:
I will do that in the next commit I push in this PR.